### ISSUE TITLE
Code Examples for Semi-Supervised Classification

### ISSUE THREAD

**Issue by lfloeer** [ericjang/gumbel-softmax#3]
Are there any code examples for the application of this kind of VAE to semi-supervised classification, as described in section 4.3 of the Gumbel-Softmax paper?
I understand the technical details in the appendix, but I'm having trouble defining the right loss function for the model depicted in Figure 6. Any help would be much appreciated!

**Comment by ericjang** [comment]
I'll try to get around to it at some point! Been pretty busy with work lately :/

**Comment by FilippoC** [comment]
Is there any news about this? I am also interested!

**Comment by Sadegh-Saberian** [comment]
Hi, Any updates on this? I am having trouble figuring out the loss function... thanks!
